input/pagecontent/terminology.md

```diff
@@ -38,6 +38,7 @@ Many value sets used in this guide are defined in the base FHIR specification, [
 |[RequestStatus](https://hl7.org/fhir/R4/valueset-request-status.html)|[AU eRequesting Diagnostic Request](StructureDefinition-au-erequesting-diagnosticrequest.html)|FHIR|
```

Comment:
Now that we bind to the new value set, this can be removed from the terminology page.